### PR TITLE
Move websockets details to own page, adjust related links

### LINKED ISSUE #236: Move websocket out of advanced.md

### PR DESCRIPTION
**Pull request by zhyatt**
Closes #236

### CODE REVIEW

File: docs/integration-guides/websockets.md [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+      "nano_3dmtrrws3pocycmbqwawk6xs7446qxa36fcncush4s1pejk16ksbmakis32c"
+    ]
+  }
+}
+```
+
+* When `all_local_accounts` is set to **`true`**, blocks that mention accounts in any wallet will be broadcasted.
+* `accounts` is a list of additional accounts to subscribe to. Both prefixes are supported.
+
+??? tip "Updating the list of accounts"
```
Comment (guilhermelawless) [inline docs/integration-guides/websockets.md]: You mentioned wanting to expand this by default, is that still the case?

File: docs/integration-guides/websockets.md [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+!!! note ""
+    Available in version 19.0+ only. When upgrading from version 18 or earlier, the node performs a confirmation height upgrade. During this process, the WebSocket notifications may include confirmations for old blocks. Services must handle duplicate notifications, as well as missed blocks as WebSockets do not provide guaranteed delivery. Reasons for missed blocks include intermittent network issues and internal containers (in the node or clients) reaching capacity.
+
+--8<-- "multiple-confirmation-notifications.md"
+
+The Nano node offers notification of confirmed blocks over WebSockets. This offers higher throughput over the HTTP callback, and uses a single ingoing connection instead of an outgoing connection for every block.
+
+The HTTP callback is still available and both mechanisms can be used at the same time.
+
+**Example**
```
Comment (guilhermelawless) [inline docs/integration-guides/websockets.md]: By moving this out of the general Advanced guide we have more header levels to play with. I think configuration and example clients should have their own (sub?)sections, and the types of messages below could be placed under a general "Specification" section.

File: docs/integration-guides/websockets.md (hunk 1 above)
Comment (zhyatt) [reply docs/integration-guides/websockets.md]: Yeah, updated

File: docs/integration-guides/websockets.md (hunk 2 above)
Comment (zhyatt) [reply docs/integration-guides/websockets.md]: Good call, made these adjustments

**Pull request merged**